python/src/apx_agent/_labeling.py

```
@@ -333,15 +343,43 @@ def align_judge(
+    # include_spans=False returns span-less objects that MemAlign cannot read.
+    # mlflow.get_trace() works on FEVM/private-link workspaces. Fixes #718.
+    thin = search_traces_for_experiment(
         experiment_id, filter_string=f"tag.{RUN_TAG} = '{run_id}'",
         return_type="list", include_spans=False,
     )
+    traces = []
+    for t in thin:
+        try:
+            traces.append(_mlflow.get_trace(t.info.trace_id))
+        except Exception:
+            pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)